test/Vendored.System.Diagnostics.DiagnosticSource.Tests/ActivitySourceTests.cs

```diff
@@ -0,0 +1,805 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.DotNet.RemoteExecutor;
+using Xunit;
+
+namespace Vendored.System.Diagnostics.Tests
+{
+    using ActivityCreationOptionsString = ActivityCreationOptions<string>;
+    using ActivityCreationOptionsContext = ActivityCreationOptions<ActivityContext>;
```

Comment:
The idea was to remove the dependencies that bring span/slice because they pull assemblies that are not available at some of the older versions of netfx.